docs/mkdocs.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+site_name: Gossamer Docs
+site_description: The Official Documentation for Gossamer | A Go Implementation of the Polkadot Host
+
+repo_name: chainsafe/gossamer
+repo_url: https://github.com/ChainSafe/gossamer
+
+# Configuration
+markdown_extensions:
+  - meta
+  - admonition
+  - codehilite
+  - toc:
+      permalink: true
+
+theme:
+  name: material
+  palette:
+    primary: pink
+    accent: indigo
+  favicon: ./assets/favicon.ico
+  features:
+    # - navigation.instant
+    - navigation.tracking
+    - navigation.sections
+    - navigation.tabs
+    - navigation.tabs.sticky
+nav:
+
+  - Getting Started:
+    - Overview: ./index.md
+    - Host Architecture: ./getting-started/overview/host-architecture.md
+    - Package Library: ./getting-started/overview/package-library.md
+    - Installation: ./getting-started/installation.md
+    - General Resources: ./getting-started/resources/general-resources.md
+    - Developer Resources: ./getting-started/resources/developer-resources.md
+  - Usage:
+    - Command Line: ./usage/command-line.md
+    - Configuration: ./usage/configuration.md
+    - Import Runtime: ./usage/import-runtime.md
+    - Import State: ./usage/import-state.md
+  - Integrate:
+    - Connect to Polkadot.js: ./integrate/connect-to-polkadot-js.md
+  - Tests: 
+    - Tests: ./testing-and-debugging/tests.md
+    - Docker Tests: ./testing-and-debugging/docker-tests.md
+    - Debugging: ./testing-and-debugging/debugging.md
+    - Logger usage: ./testing-and-debugging/logger-usage.md
```

Comment:
Update this to the following:

Test Suite
Debugging

Reply:
@dutterbutter 

Do you mean for me to remove two entries or rename two entries and if so which ones?

Reply:
No need to have 4 individual sections, reduce it to the following

### Testing and Debugging

**Test Suite**
**Debugging**

Reply:
@dutterbutter Sweet, I dont see content on the debugging page at the moment
https://hackmd.io/bfVYAP2gTe6bjv9TMJ8tqg

Reply:
As mentioned in standup use the Logger hackmd link

Reply:
https://hackmd.io/qxsxNhGQQGWYTLfMz5_Oyw